sr_robot_launch/launch/load_robot_description_bimanual.launch

```diff
@@ -57,10 +57,9 @@
   <arg name="arm_2_z" default="0.0"/>
 
   <arg if="$(eval arms and arg('hands') == 'none')" name="robot_description" default="'$(find sr_multi_description)/urdf/bimanual_ur.urdf.xacro'"/>
-  <arg if="$(eval arms and not arg('hands') == 'none')" name="robot_description" default="'$(find sr_multi_description)/urdf/bimanual_srhand_ur.urdf.xacro'"/>
-
+  <arg if="$(eval arms and not arg('hands') == 'none')" name="robot_description" default="'$(find sr_multi_description)/urdf/bimanual_srhand_ur.urdf.xacro'"/>  
   <arg if="$(eval arms and arg('hands') == 'none')" name="load_robot_description_command" value="xacro $(arg robot_description) robot_model:=$(arg robot_model) arm_1_z:=$(arg arm_1_z) arm_2_z:=$(arg arm_2_z) arm_x_separation:=$(arg arm_x_separation) arm_y_separation:=$(arg arm_y_separation)"/>
-  <arg if="$(eval arms and not arg('hands') == 'none')" name="load_robot_description_command" value="xacro $(arg robot_description) robot_model:=$(arg robot_model) right_hand_type:=$(arg right_hand_type) right_hand_version:=$(arg right_hand_version)
+  <arg if="$(eval arms and not arg('hands') == 'none')" name="load_robot_description_command" value="xacro $(arg robot_description) hands:=$(arg hands) robot_model:=$(arg robot_model) right_hand_type:=$(arg right_hand_type) right_hand_version:=$(arg right_hand_version)
```

Comment:
I don't understand why only the right hand is passed here?

Reply:
I also pass to the xacro now the hands argument and so if hands=right, although the left arguments are passed, they will not be used inside the xacro. So I think that condition works for both, right and left cases (if I'm not wrong). In my opinion adding the if both... would make the code longer, but less confusing indeed.

Reply:
yes, this sounds good